pkg/client/client.go

```diff
@@ -184,10 +185,10 @@ func New() (Client, error) {
 }
 
 func (c *AxieClient) createTransactOps(ctx context.Context, privateKeyStr string) (*bind.TransactOpts, error) {
-	return c.createTransactOpsWithGasLimitRange(ctx, privateKeyStr, 165313, 198888)
+	return c.createTransactOpsWithGasLimitRange(ctx, privateKeyStr, 1000000)
 }
 
-func (c *AxieClient) createTransactOpsWithGasLimitRange(ctx context.Context, privateKeyStr string, lowerGasLimit int64, upperGasLimit int64) (*bind.TransactOpts, error) {
+func (c *AxieClient) createTransactOpsWithGasLimitRange(ctx context.Context, privateKeyStr string, gasLimit uint64) (*bind.TransactOpts, error) {
```

Comment:
rename to `createTransactOpsWithGasLimit`?
or do we still need this function? can just merge with `createTransactOps`?

Reply:
done